packages/flame/lib/src/geometry/circle_component.dart

```diff
@@ -83,6 +83,11 @@ class CircleComponent extends ShapeComponent {
         scaledRadius * scaledRadius;
   }
 
+  @override
+  bool containsLocalPoint(Vector2 point) {
```

Comment:
Any reason why this is implemented for this geometry component, but not the others?

Reply:
Only because I used a `CircleComponent` in a test, so I had to make it work. But I can add an implementation for a `PolygonComponent` too.

Reply:
done